.travis.yml

```diff
@@ -47,6 +47,20 @@ matrix:
            PROJECT_URL=https://github.com/joblib/joblib.git
       python: 3.7
       if: commit_message =~ /(\[ci downstream\]|\[ci joblib\])/
+    - os: linux
+      env: PROJECT=ray
+      python: 3.7
+      if: commit_message =~ /(\[ci downstream\]|\[ci ray\])/
+      install:
+        - pip install --upgrade -r dev-requirements.txt
+        - pip install setproctitle psutil ray==0.6.4
+        - PROJECT_DIR="$(python -c "import os,ray,sys;print(os.path.dirname(ray.__file__));sys.stdout.flush()")"
+        - rm $PROJECT_DIR/cloudpickle/cloudpickle.py
+        - cp cloudpickle/cloudpickle.py $PROJECT_DIR/cloudpickle/cloudpickle.py
+        - pip list
+      script:
+        - COVERAGE_PROCESS_START="$TRAVIS_BUILD_DIR/.coveragerc" PYTHONPATH='.:tests' pytest -r s
+        - pytest -vl --duration=10 $PROJECT_DIR/tests/test_mini.py
```

Comment:
Wouldn't it make sense to run more tests? I think it's fine if running the ray tests takes 20 min or so because we only run them when we use the specific `[ci downstream/ray]` commit message flags, typically as a final check in a big pull request refactoring.

flaky tests on the other hand is a real issue.

Reply:
I agree, however, some of the Ray tests are flaky and can't be added. I plan to add more tests once they are not flaky.

Reply:
I picked out a few for now that are not flaky and that hopefully hit more cloudpickle codepaths.